stat/stat.go

```diff
@@ -19,7 +19,10 @@ const (
 	// https://en.wikipedia.org/wiki/Quantile#Estimating_the_quantiles_of_a_population
 
 	// Empirical treats the distribution as the actual empirical distribution.
-	Empirical CumulantKind = 1
+	Empirical CumulantKind = 1 + iota
```

Comment:
I don't think `iota` is appropriate here. The values we are using match the quantile types provided by R and are not just a sequence of enumerated values.

Would it be worth linking to the table in https://en.wikipedia.org/wiki/Quantile#Estimating_quantiles_from_a_sample if this list gets longer?

Reply:
done.